### PR TITLE
feat(scoop-import): Import a Scoop installation from JSON

### LINKED ISSUE #3934: [Feature request] Import command
I think it would be nice to have a counterpart to `scoop export > file`, for example `scoop import file`. I know this can currently be achieved in an [alternative way][1], but still it would be more convenient to have a built-in command.

[1]: https://github.com/lukesampson/scoop/issues/1543#issuecomment-308894312

### PR DESCRIPTION
**Pull request by rashil2000**
#### Description
Awesome new feature of Scoop! 🎉🎊

Export to and import a Scoop installation from a JSON file.

#### Motivation and Context

Closes #3934 

#### How Has This Been Tested?

```
scoop export -c > Desktop\scoopfile.json
scoop import Desktop\scoopfile.json
```

#### Checklist:

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).
- [x] I have ensured that I am targeting the `develop` branch.
- [x] I have updated the documentation accordingly.
- [ ] I have updated the tests accordingly.
- [x] I have added an entry in the CHANGELOG.

### CODE REVIEW

File: libexec/scoop-import.ps1 [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Usage: scoop import <path/url to scoopfile.json>
+# Summary: Imports apps, buckets and configs from a Scoopfile in JSON format
+
+param([Parameter(Mandatory)][String]$scoopfile)
+
+. "$PSScriptRoot\..\lib\manifest.ps1"
```
Comment (niheaven) [inline libexec/scoop-import.ps1:6]:
```suggestion
. "$PSScriptRoot\..\lib\manifest.ps1"

param(
    [Parameter(Mandatory)]
    [String]
    $scoopfile
)
```

**Review by niheaven — COMMENTED** [review]
I didn't test the import feature, but it's okay in static reviews.

**Comment by rashil2000** [comment]
@niheaven anything remaining here?

**Review by niheaven — APPROVED** [review]

**Comment by zStruCat** [comment]
One minor issue
Regarding
`scoop export -c > Desktop\scoopfile.json`
"-c" option seems not covered in scoop help export. 
I presume it means create? But the command does create a file without "-c".

**Comment by rashil2000** [comment]
Thanks for noticing @zStruCat! It was a typo and I've fixed it

**Pull request merged**